test/jdk/ProblemList.txt

```diff
@@ -460,6 +460,15 @@ java/awt/image/multiresolution/MultiresolutionIconTest.java 8291979 linux-x64,wi
 java/awt/event/SequencedEvent/MultipleContextsFunctionalTest.java 8305061 macosx-x64
 sun/java2d/DirectX/OnScreenRenderingResizeTest/OnScreenRenderingResizeTest.java 8301177 linux-x64
 
+# Several tests which fail on some hidpi systems/macosx12-aarch64 system
+java/awt/Window/8159168/SetShapeTest.java 8274106 macosx-aarch64
+java/awt/image/multiresolution/MultiResolutionJOptionPaneIconTest.java 8274106 macosx-aarch64
+
+java/awt/Mouse/EnterExitEvents/DragWindowTest.java 8298823 macosx-all
+java/awt/Focus/NonFocusableWindowTest/NonfocusableOwnerTest.java 8280392 windows-x64
+java/awt/Mixing/AWT_Mixing/OpaqueOverlapping.java 8294264 windows-x64
+
+java/awt/dnd/MissingDragExitEventTest/MissingDragExitEventTest.java 8288839 windows-x64
```

Comment:
These tests are excluded on Windows, they don't seem to belong under the heading provided.

Should they rather be placed closer to similar tests, in alphabetical order?

Reply:
There was a line gap which should signify these tests doesn't belong to that heading, but anyways I have placed those tests closer to similar tests..

Reply:
Thanks.

I can't find any guidelines where newly added tests should be placed. Looking at the list, I have the impression that it was common to add tests to the same group, keeping the list sorted alphabetically.

However, as far as I can see, most new lines are added to an end of a section. This is easier, I guess…